app/controllers/katello/api/v2/packages_controller.rb

```diff
@@ -12,10 +15,23 @@ def index
       super
     end
 
-    private
+    def auto_complete_name
+      rpms = Rpm.in_repositories(@repositories)
+      col = "#{Rpm.table_name}.name"
+      rpms = rpms.where("#{Rpm.table_name}.name ILIKE ?", "#{params[:term]}%").select(col).group(col).order(col).limit(20)
```

Comment:
Maybe make it a constant on the autocomplete module (https://github.com/Katello/katello/blob/fc47f03eaceef0975598efdc542acf6976724426/app/controllers/katello/concerns/filtered_auto_complete_search.rb) since that typically gets included everywhere and could connect the number to autocomplete?